index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Additional-1</title>
+  <link rel="stylesheet" href="./Styles/style.css">
+</head>
+
+<body>
+  <header>
+    <h1>Header Block</h1>
+  </header>
+  <main>
+    <h2>Main block with content</h2>
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni alias vero quam quia veniam harum eos? Aliquid
+      debitis, modi rem excepturi vero, delectus optio quibusdam laboriosam alias maiores architecto. Iure.</p>
+  </main>
+  <aside>
+    <h2>Sidebar with content</h2>
+  </aside>
+  <footer>
+    <h1>Footer Block</h1>
```

Comment:
h1 should be only one on-page, if you had it on header, don't use it on footer again, you can use anyone else headline but not h1